### PR TITLE
Tweaks to "CIDv1/Base32 Migration" section in 0.4.18 release notes.

### PR DESCRIPTION
**Pull request by kevina**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -224,8 +224,9 @@ case intensive encoding required by domain names. This command converts a CID to
 version 1 and encodes it using base32.
 
 (2) A hack to allow locally looking up blocks associated with a CIDv0 CID using
-the equivalent CIDv1 CID. This hack will eventually (likely in the next release)
-be replaced with a proper CID-version agnostic datastore.
+the equivalent CIDv1 CID (or the reverse). This hack will eventually (likely in the next release)
```
Comment (kevina) [inline CHANGELOG.md]: @Stebalien 

> (likely in the next release)

That may be a little optimization considering that are likely going to need to make changes in libp2p.

File: CHANGELOG.md [hunk 2]
```diff
@@ -224,8 +224,9 @@ case intensive encoding required by domain names. This command converts a CID to
...
+be replaced with a multihash indexes blockstore, which is agnostic to both the
```
Comment (Stebalien) [inline CHANGELOG.md]: "multihash indexed"

File: CHANGELOG.md (hunk 1 above)
Comment (Stebalien) [reply CHANGELOG.md]: Yeah, maybe. Feel free to remove it.

Comment (kevina) [reply CHANGELOG.md]: Okay. Done.

**Pull request merged**